src/mmtk.rs

```diff
@@ -432,7 +432,10 @@ impl<VM: VMBinding> MMTK<VM> {
                 .store(true, Ordering::Relaxed);
             self.gc_requester.request();
             VM::VMCollection::block_for_gc(tls);
+            return true;
```

Comment:
The user only gets the return value after the GC is finished. Is that still useful?

Reply:
I mean that's how it should be right? I'm not sure I understand your question.

Reply:
OK. Maybe document this more cleraly. See my other comment.

Reply:
Done.